helm/hpcc/templates/_helpers.tpl

```diff
@@ -761,7 +761,15 @@ A kludge to ensure mounted storage (e.g. for nfs, minikube or docker for desktop
 # This is only required when mounting a remote filing systems from another container or machine.
 # NB: this includes where the filing system is on the containers host machine .
 # Examples include, minikube, docker for desktop, or NFS mounted storage.
-{{- $permCmd := printf "chown -R %v:%v %s || true" .uid .gid .volumePath }}
+{{- $permCmd := "" -}}
+{{- $uid := .uid -}}
+{{- $gid := .gid -}}
+{{- range $index, $volume := .volumes }}
+ {{- if ne $index 0 }}
+  {{- $permCmd = printf "%s & " $permCmd -}}
```

Comment:
Could this container terminate if the later chown succeeds, but the first are still running?

Reply:
good question, I think not, I test with earlier chown's failing and future ones succeeding,
but not laters failing/succeeding/ while earlier still running.

I'll circle back and check when I get back.